docs/en-us/sidecarset.md

```diff
@@ -340,6 +340,10 @@ The above is the complete hot upgrade process. If a Pod needs to be hot upgraded
 
 ![sidecarset hotupgrade](/img/docs/sidecarset_hotupgrade.png)
 
+#### Migration Demo
+The SidecarSet hot upgrade mechanism not only completes the switching of mesh containers and provides a coordination mechanism for old and new versions, but this is only the first step in a long journey. The mesh container also needs to provide a PostStartHook script to complete the hot migration of the mesh service itself (the above Migration process), such as Envoy hot restart and Mosn lossless restart.
```

Comment:
The SidecarSet hot upgrade mechanism not only completes the switching between mesh containers，but also provides a coordination mechanism for old and new versions. Yet this is only the first step of a long journey.